app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
Your code will crash if command not have 3 elements

Reply:
**Note: Check your code using this [checklist](https://github.com/mate-academy/py-copy-file/blob/master/checklist.md) before pushing your solution**

Reply:
Do I need to use a for cycle?

Reply:
No, you need to check that len of command == 3

Reply:
add more declarative variable names for command[1] and  command[2]
I don't even realize what is it